hathor/merged_mining/coordinator.py

```diff
@@ -741,15 +754,10 @@ def job_request(self) -> None:
             self.send_request('mining.notify', job.to_stratum_params())
 
             # for debugging only:
-            bitcoin_block = job.build_bitcoin_block_header(self.dummy_work(job))
+            bitcoin_block = job.build_bitcoin_block_header(job.dummy_work())
```

Comment:
Shouldn't this line be a comment?

Reply:
Maybe, but it isn't hurting performance and I think is very useful to be able to turn it on just by using `--debug`, which I think can't be verified right now just by looking at the `log` object. And I don't think it's worth propagating a `debug` param, since it would only be used here without clear benefit.

Reply:
Ok! It makes sense. It is a shame we can't check the log level at the `log` object, but I also understand why it is done this way.